bindings/wasm/test/fixtures/unitSphereGLTF.mjs

```diff
@@ -0,0 +1,10 @@
+import {GLTFNode, Manifold} from '../../lib/manifoldCAD';
+
+// Models that build GLTF objects need not export their results.  The
+// scene builder will track GLTF nodes as they are created.  Essentially,
+// it's possible to build a manifoldCAD model as a side-effect of an import.
```

Comment:
Hmm, thanks for noting this. Is this desirable behavior? If I make a library, I expect to e.g. `export function gear(params...)` and `const result gear(example params)`. On import, it shouldn't generate any geometry, just make the function available, which I believe is how your PR works now, right? If I instead switch the example result to a `new GLTFNode().manifold = gear(example params)`, I would expect it to work the same way, not pollute my node structure in the place I'm importing it with an example I have no way to remove. The node should only appear if I run the library directly as the top-level file.

Reply:
At least on my stuff, feel free to comment.  More brain cells are good.

Reply:
I like the sound of this plan quite a bit. From the tense I'm having a little trouble determining how much of this is how it currently works in this PR and how much is a change to still be made?